students/eunhyekim/users/views.py

```diff
@@ -37,10 +37,13 @@ def post(self, request):
             if User.objects.filter(email = email).exists():
                 return JsonResponse({"MESSAGE":"User_Already_Exists"}, status = 400)
 
+            hashed_password = bcrypt.hashpw(data["password"].encode('utf-8'), bcrypt.gensalt())
+            decoded_hashed_password = hashed_password.decode('utf-8')
```

Comment:
어차피 디코딩 하실거라면, 두 번에 걸쳐서 하실 필요가 없어보입니다!
```suggestion
            hashed_password = bcrypt.hashpw(data["password"].encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Reply:
수정하였습니다!